roles/addons/slurm/tasks/main.yml

```diff
@@ -75,7 +75,7 @@
     name: slurmctld
     state: started
     enabled: no         # Dangerous to enable slurm controller...
-  when: slurm_profile == 'controler'
+  when: slurm_profile == 'controller'
```

Comment:
To avoid breaking existing configuration, we can consider `when: slurm_profile == 'controller' or slurm_profile == 'controler'` with a deprecation notice.